src/instructlab/model/train.py

```diff
@@ -355,19 +355,19 @@ def clickpath_setup(is_dir: bool) -> click.Path:
 @click.option(
     "--phased-phase2-num-epochs",
     cls=clickext.ConfigOption,
-    type=click.IntRange(min=1),
+    type=click.IntRange(min=0),
     help="Number of epochs to run for second phase of end-to-end training.",
 )
 @click.option(
     "--phased-phase2-samples-per-save",
     cls=clickext.ConfigOption,
-    type=click.IntRange(min=1),
+    type=click.IntRange(min=-1),
```

Comment:
Should this have a minimum of -1 if the minimum in the corresponding field is 0?

Reply:
my bad, fixed